xclim/indices/generic.py

```diff
@@ -646,3 +646,29 @@ def extreme_temperature_range(
     u = str2pint(low_data.units)
     out.attrs["units"] = pint2cfunits(u - u)
     return out
+
+
+def degree_days(tas: xr.DataArray, thresh: str, condition: str) -> xr.DataArray:
```

Comment:
I'd suggest making this a fully working indice. i.e. : add a `@declare_units` decorator and convert units with `to_agg_units` at the end.